src/app/public/modules/shared/image-resources.module.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  SKY_LIB_RESOURCES_PROVIDERS
+} from '@skyux/i18n';
+
+import {
+  SkyImageResourcesProvider
+} from '../../plugin-resources/image-resources-provider';
+
+@NgModule({
+  providers: [{
+    provide: SKY_LIB_RESOURCES_PROVIDERS,
+    useClass: SkyImageResourcesProvider,
+    multi: true
+  }]
+})
+export class SkyImageResourcesModule { }
```

Comment:
Since this will be used by all modules in this library, we should name it `SkyMediaResourcesModule`.